site/content/docs/concepts/pipelines.en.md

```diff
@@ -103,7 +103,7 @@ You can see every available configuration option for `manifest.yml` on the [pipe
 
 There are 3 main parts of this file: the `name` field, which is the name of your pipeline, the `source` section, which details the repository and branch to track, and the `stages` section, which lists the environments you want this pipeline to deploy to. You can update this anytime, but you must commit and push the changed files and run `copilot pipeline deploy` afterwards.
 
-Typically, you'll update this file if you add new environments you want to deploy to, or want to track a different branch. If you are using CodeStar Connections to connect to your repository and would like to utilize an existing connection rather than let Copilot generate one for you, you may add the connection name here. The pipeline manifest is also where you may add a manual approval step before deployment or commands to run tests (see "Adding Tests," below) after deployment.
+Typically, you'll update this file to add environments to deploy to, specify the order of deployments, add actions for the pipeline to run before or after deployment, or change the branch to track. You may add a manual approval step before deployment or commands to run tests (see "Adding Tests and Other Actions," below). If you are using CodeStar Connections to connect to your repository and would like to utilize an existing connection rather than let Copilot generate one for you, you may add the connection name here.
```

Comment:
```suggestion
Typically, you'll update this file to add environments to deploy to, specify the order of deployments, add actions for the pipeline to run before or after deployment, or change the branch to track. You may add a manual approval step before deployment or commands to run tests (see [Adding Tests and Other Actions](#link here)). If you are using CodeStar Connections to connect to your repository and would like to utilize an existing connection rather than let Copilot generate one for you, you may add the connection name here.
```

Reply:
already outdated, sorry!